packages/react-app-scaffolder/app/templates/base/src/core/index.tsx

```diff
@@ -19,6 +20,12 @@ window.reapit = {
 
 export const renderApp = (Component: React.ComponentType) => {
   const rootElement = document.querySelector('#root') as Element
+  const isDesktop = getMarketplaceGlobalsByKey()
```

Comment:
Ensure 16px rem base is applied and does not break layouts
Apply helper to change rem base to 14px when in desktop